packages/react-native-bridge/android/build.gradle

```diff
@@ -341,3 +364,62 @@ If they are changed, the isBundleUpToDate flag is switched to false. That flag i
         delete nodeModulesFolders
     }
 }
+
+// Allows to override the artifact version by passing a -PbintrayVersion
+// parameter when calling Gradle
+def getBintrayVersion() {
+  return project.properties['bintrayVersion'] ?: android.defaultConfig.versionName
+}
+
+bintray {
+    user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
+    key = project.hasProperty('bintrayKey') ? project.property('bintrayKey') : System.getenv('BINTRAY_KEY')
+    publications = ['GutenbergMobilePublication']
+    publish = true
+    pkg {
+        repo = 'maven'
+        name = 'react-native-gutenberg-bridge'
+        licenses = ['GPL-2.0']
+        userOrg = 'wordpress-mobile'
+        vcsUrl = 'https://github.com/wordpress-mobile/gutenberg-mobile.git'
```

Comment:
In Oguz's and @mchowning original prototype, the question was raised of whether to use WordPress Bintray organization and Gutenberg as the VCS repository. I like this idea because it moves closer to the logic existing without the need to be a submodule of `gutenberg-mobile` itself.